Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -282,4 +289,17 @@ const std::string GameListItem::GetWiiFSPath() const
 
 	return ret;
 }
-
+bool GameListItem::IsElfOrDol() const
+{
+	//This can probably be made cleaner - The way elf/dol support in the gamelist
+	//is implemented isn't the greatest.
+	const std::string name = GetName();
+	const size_t pos = name.rfind('.');
+
+	if (pos != std::string::npos)
+	{
+		return name.substr(pos) == ".elf" ||
+			name.substr(pos) == ".dol";

```

Comment:
I don't know what you mean by one space less, do I need a new line between functions? I noticed I was missing one and it looked weird so I just corrected that